woocommerce/woocommerce-android
# Define order creation experimental toggle

Summary
==========
Fixes issue #5285 by creating the experimental toggle for activating the Order Creation feature.

Screenshots
==========
| Beta feature section description | Order Creation toggle |
| ------------- | ------------- |
| ![Screenshot_20211124_043733](https://user-images.githubusercontent.com/5920403/143195433-eb7d0afb-e0a4-456b-bd43-678290f37620.png) | ![Screenshot_20211124_043739](https://user-images.githubusercontent.com/5920403/143195452-678fd98d-6515-4c23-b6c1-32dee8e41792.png) |

How to Test
==========
1. Go to the App settings and scroll down to the Beta Features section, verify that the description shows at least the `Order creation` text
2. Click the Beta Features section row and verify if the list of toggles there matches the section row description on the previous view
3. Interact with the Order Creation toggle and verify that changing it persists the changes as expected.



Update release notes:
